src/GitLinkTask/GitLink.cs

```diff
@@ -10,21 +10,18 @@ namespace GitLinkTask
     using System;
     using System.IO;
     using System.Linq;
-    using GitLink;
-    using GitLink.Pdb;
-    using GitLink.Providers;
+    using global::GitLink;
+    using global::GitLink.Pdb;
+    using global::GitLink.Providers;
     using Catel.Logging;
     using Microsoft.Build.Framework;
     using Microsoft.Build.Utilities;
 
-    public class LinkProject : Task
+    public class GitLink : Task
```

Comment:
I've never seen an MSBuild task whose class did not match the name of the xml tag. There is a strong precedent to find the class based on the tag name. As this is in its own assembly and namespace anyway, why the concern? 
I'm not too fond of either name, actually. Something more descriptive like LinkPdbToGitRemote would be my preference. Any objections if I rename it to that?